02-wrangling.Rmd

```diff
@@ -129,11 +142,11 @@ knitr::include_graphics("img/tidy_data.jpeg")
 
 ### Why is tidy data important in R?
 
-First, one of R's most popular plotting tool sets, the `ggplot2` package (which is one of the packages that the `tidyverse` package loads), expects the data to be in a tidy format. Second, most statistical analysis functions also expect data in a tidy format. Given that both of these tasks are central in almost all data analysis projects, it is well worth spending the time to get your data into a tidy format upfront. Luckily there are many well-designed `tidyverse` data cleaning/wrangling tools to help you easily tidy your data. Let's explore them now!
+First, one of R's most popular plotting toolsets, the `ggplot2` package (which is one of the packages that the `tidyverse` package loads), expects the data to be in a tidy format. Second, most statistical analysis functions also expect data in a tidy format. Given that both of these tasks are central in almost all data analysis projects, it is well worth spending the time to get your data into a tidy format upfront. Luckily there are many well-designed `tidyverse` data cleaning/wrangling tools to help you easily tidy your data. Let's explore them now!
```

Comment:
This seems redundant to the text in 3.4. Should we merge this with that text and remove this?

Or maybe we start off 3.4 asking a question - such as, is there a best way to organize your data for all/most analyses? And then say, yes indeed there does seem to be a standard format that is almost universally useful, it is called the tidy data format. We will first explain what it is, and then why it's so effective.

Then here, we have all the rationale for why. 

What do you think? The reason I am suggesting this, is that it's hard to fully justify why we are telling them to use something they have not yet even seen. If you prefer moving this all to 3.4 though, I am can be happy with that too. The biggest issue is that we need to remove this redundancy on the why.

Reply:
Good point! I removed the redundancy in section 3.4 and moved the "why" up before the "what". Not sure if it'd be better to do it the other way around? (what and then why?)